server/games/game_results.py

```diff
@@ -75,21 +75,38 @@ def outcome(self, army: int) -> GameOutcome:
         if army not in self:
             return GameOutcome.UNKNOWN
 
-        outcomes = set(
-            r.outcome for r in self[army]
-            if r.outcome is not GameOutcome.UNKNOWN
+        voters = defaultdict(set)
```

Comment:
Hm, I see your point, but then you would need
```python
voters[outcome] = voters.get(outcome, set()).add(reporter)
```
instead of 
```python
voters[outcome].add(reporter)
```
which I think is a bit less slick?

Reply:
Let's see if it is sick.
First of all you know what's inside,, because near default dict you don't have proper typing.
Second, you don't add unnecessary magic, everything is explicit.

Reply:
Also @norraxx solution will create extra sets for ever call whereas `defaultdict` only creates them when they're needed. IMO `defaultdict` is the way to go. This is exactly what `defaultdict` was created for.